README.md

```diff
@@ -1,5 +1,7 @@
 # Uphold API Reference
+
```

Comment:
Is there a way to enforce this? :smiley:

Reply:
Yes, I thought about that too :) I am planning to introduce a markdown linter, but that's for another PR ;)